src/main/resources/edu/cornell/kfs/concur/cu-ojb-concur.xml

```diff
@@ -0,0 +1,20 @@
+<!--
```

Comment:
since this file doesn't actually include any OJB descriptors is it necessary to include at this time?